From e27d130644032b62b6ab1cb9a4622e4a99c0cb3d Mon Sep 17 00:00:00 2001
From: Benjamin Kaduk <bkaduk@akamai.com>
Date: Tue, 7 Sep 2021 09:17:56 -0700
Subject: [PATCH 51/60] QUIC: expound on what DoS attacks QUIC avoids

The limit on the amount of queued data is to avoid being an amplification
vector, specifically.
---
 doc/man3/SSL_CTX_set_quic_method.pod | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/man3/SSL_CTX_set_quic_method.pod b/doc/man3/SSL_CTX_set_quic_method.pod
index b2d78872ca..1ed076c301 100644
--- a/doc/man3/SSL_CTX_set_quic_method.pod
+++ b/doc/man3/SSL_CTX_set_quic_method.pod
@@ -139,8 +139,8 @@ generating their own errors.
 
 See L<https://tools.ietf.org/html/RFC9001#section-4.1> for more details.
 
-To avoid DoS attacks, the QUIC implementation must limit the amount of data
-being queued up. The implementation can call
+To avoid amplifying DoS attacks, the QUIC implementation must limit the amount
+of data being queued up. The implementation can call
 SSL_quic_max_handshake_flight_len() to get the maximum buffer length at each
 encryption level.
 
-- 
2.40.0.rc0

